packages/functions/src/context.ts

```diff
@@ -50,9 +49,11 @@ export class ContextProvider {
     try {
       // HACK: Until we have a separate instanceId package, this is a quick way
       // to load in the messaging instance for this app.
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Do we still not have a separate instance id package we could depend on?

Reply:
Yes, we have `installations` package which is the replacement for IID. I'm not sure if we can start using it in functions. We should talk about it. @mmermerkaya

Reply:
Let's punt that to a future PR anyway.